src/controllers/QueryRunner.ts

```diff
@@ -281,11 +279,11 @@ export default class QueryRunner {
                         // Iterate over the rows to paste into the copy string
                         for (let rowIndex: number = 0; rowIndex < result.resultSubset.rows.length; rowIndex++) {
                             let row = result.resultSubset.rows[rowIndex];
-                            let cells = row.slice(range.fromCell, (range.toCell + 1));
-                            if (self.shouldRemoveNewLines()) {
-                                // Remove all new lines from cells
-                                cells = cells.map(x => self.removeNewLines(x));
-                            }
+                            let cellObjects = row.slice(range.fromCell, (range.toCell + 1));
+                            // Remove newlines if requested
+                            let cells = self.shouldRemoveNewLines()
+                                ? cellObjects.map(x => self.removeNewLines(x.displayValue))
```

Comment:
This looks good here. But to verify to others, this now correctly parses an empty array in the case where we are rendering the placeholder values?